document/static/js/modules/schema/common.js

```diff
@@ -144,6 +150,37 @@ export const figure = {
         dom.dataset.figureCategory = node.attrs.figureCategory
         dom.dataset.caption = node.attrs.caption
         dom.id = node.attrs.id
+        dom.aligned = node.attrs.aligned
```

Comment:
I think you mean `dom.align`? Anyway, this shouldn't be used. You should only add the class (as you do below) and then do the alignment by means of CSS. `dom.align` is no longer part of HTML5. See https://www.w3schools.com/tags/att_object_align.asp

Reply:
ok.. the idea was to store in the dom the attribute 'aligned' for the parseDOM functionality. 
now i ve commented that line both in parseDOM and toDOM functionalities.. shall we do something for parseDOM functionalities ?